contracts/interchain-token-service/src/types.rs

```diff
@@ -0,0 +1,50 @@
+use soroban_sdk::{contracttype, Bytes, BytesN, String};
+
+#[contracttype]
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+#[repr(u32)]
+pub enum MessageType {
+    InterchainTransfer = 0,
+    DeployInterchainToken = 1,
+    DeployTokenManager = 2,
+    SendToHub = 3,
+    ReceiveFromHub = 4,
+}
+
+#[contracttype]
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub enum TokenManagerType {
```

Comment:
Is there any reason not to assign values explicitly for TokenManagerType? If not, let's assign values from 0 to 4, similar to MessageType. [reference](https://github.com/axelarnetwork/interchain-token-service/blob/19d27a4a5c3b72028fa3a6fcad64c70c3e2b0e21/test/constants.js#L10)